code/game/atoms_movable.dm

```diff
@@ -6,15 +6,19 @@
 	// var/elevation = 2    - not used anywhere
 	var/move_speed = 10
 	var/l_move_time = 1
-	var/throwing = 0
-	var/thrower
-	var/turf/throw_source = null
-	var/throw_speed = 2
+	var/datum/thrownthing/throwing = null
+	var/throw_speed = 2 //How many tiles to move per ds when being thrown. Float values are fully supported
 	var/throw_range = 7
-	var/no_spin_thrown = 0 //set this to 1 if you don't want an item that you throw to spin, no matter what. -Fox
+	var/no_spin = 0
+	var/no_spin_thrown = 0

```

Comment:
It's controlled based on the throw_at call, there's no way to override it per-object. There wasn't any real reason to introduce feature loss.